### PR TITLE
[IMP] web: search view for "Schedule activity" dialog

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/mail/static/src/js/views/activity/activity_view.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ const ActivityView = BasicView.extend({
     init: function () {
         this._super.apply(this, arguments);
 
+        const { search_view_id } = arguments[1].action;
```
Comment (aab-odoo) [inline addons/mail/static/src/js/views/activity/activity_view.js:32]: suggestion: 
```js
    init: function (viewInfo, params) {
        // ...
        const { search_view_id } = params.action || {};
```

**Review by aab-odoo — COMMENTED** [review]
Seems good :+1:

**Comment by Polymorphe57** [comment]
@aab-odoo Ping Pang Poung!

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**